src/Pagerfanta/Pagerfanta.php

```diff
@@ -498,4 +499,24 @@ private function needsToIntegerConversion($value)
     {
         return (is_string($value) || is_float($value)) && (int) $value == $value;
     }
+
+    /**
+     * Get page number of the item at specified position (1-based index)
+     *
+     * @param integer $position
+     *
+     * @return integer
+     */
+    public function getPageNumberForItemAtPosition($position)
+    {
+        if (!is_int($position)) {
+            throw new NotIntegerItemException();
```

Comment:
The name of this exception doesn't match so well now we've renamed things.  I suggest making it just `NotIntegerException` as then it's more generic - I don't think we'd ever use `NotIntegerPositionException` anywhere else!